packages/block-library/src/navigation/index.php

```diff
@@ -86,30 +86,100 @@ function block_core_navigation_build_css_font_sizes( $attributes ) {
 }
 
 /**
- * Renders a Navigation Block derived from data from the theme_location assigned
- * via the block attribute 'location'.
+ * Returns the menu items for a WordPress menu location.
  *
- * If no location was provided as a block attribute then false is returned.
- *
- * @param  string $location The location of the classic menu to display.
- * @param  object $attributes The block attributes.
- * @return string|false HTML markup of a generated Navigation Block or false if no location is specified.
+ * @param string $location The menu location.
+ * @return array Menu items for the location.
  */
-function gutenberg_render_menu_from_location( $location, $attributes ) {
+function gutenberg_get_menu_items_at_location( $location ) {
 	if ( empty( $location ) ) {
-		return false;
+		return;
```

Comment:
I think it might better to return an empty array when "bailing" early in this function. This way, we can have a single return type.